chart/values.yaml

```diff
@@ -260,7 +260,7 @@ csi:
 
 io_engine:
   # -- Log level for the io-engine service
-  logLevel: info,io_engine=info
```

Comment:
This change requires additional instructions for upgrade, Is this necessary for the 2.0 branch?